.github/workflows/e2e-tests-with-operator.yml

```diff
@@ -209,4 +209,4 @@ jobs:
       success: ${{  needs.run-batch-job.result == 'success' }}
       region: us-west-2
     secrets:
-      roleArn: ${{ secrets.METRICS_ROLE_ARN }}
\ No newline at end of file
+      roleArn: ${{ secrets.METRICS_ROLE_ARN }}
```

Comment:
I don't know why this happened, I used the Git editor and it just added it automatically.

Reply:
The previous revision had no `\n` at `EOF`, which makes it an invalid text file (per the POSIX spec, most things don't really care).  That's indicated by the line-in-circle symbol in the diff.  The editor is doing the right thing.